Automattic/newspack-theme
# feat: enable jetpack share buttons on pages, too

### All Submissions:






### Changes proposed in this Pull Request:



Allows Jetpack's share buttons to appear on pages. Currently they're only allowed to display on singular posts of any post type. This comes from a request by a few publishers who want to display these buttons on their About pages, so please let me know if this was an intentional decision.

### How to test the changes in this Pull Request:

1. In WP admin, go to **Jetpack > Settings > Sharing**. In the "Sharing buttons" section, turn on the "Add sharing buttons to your posts and pages" option and configure sharing buttons so at least a few are enabled.
2. On `master`, view posts of various types using all the available templates and note how the buttons appear. Then view a page and observe that no share buttons are displayed.
3. Check out this branch and refresh the posts and the page. Confirm that a.) the share buttons appear on posts exactly as they did while on `master`, and b.) the share buttons now appear on pages as well.

### Other information:
